src/index.html

```diff
@@ -7,6 +7,376 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="wrapper">
+      <header class="header wrapper__header">
+        <div class="header__container">
+          <a href="#" class="header__logo">
+            <img src="./images/logo.png" alt="logo">
+          </a>
+
+          <nav class="header__menu menu-header">
+            <ul class="menu-header__list">
+              <li class="menu-header__item">
+                <a href="#apple" class="menu-header__link menu-header__link--is-active is-active">Apple</a>
```

Comment:
You added an `is-active` class, but I don't see it in the styles. Do you really need him here?